docs/user-manual/en/protocols-interoperability.md

```diff
@@ -215,9 +215,11 @@ The first is the 5.x style destination filter that identifies the destination as
 The second identifies the number of ```paths``` that identify the consumer queue such that it can be parsed from the
 destination.
 For example, the default 5.x virtual topic with consumer prefix of ```Consumer.*.```, would require a
-```virtualTopicConsumerWildcards``` filter of:
+```virtualTopicConsumerWildcards``` filter of ```Consumer.*.>;2```. As url parameter this transforms to ```Consumer.*.%3E%3B2``` when
+the url significant characters ```;,``` are escaped with their hex code points. 
```

Comment:
should there be ```>;``` instead of ```;,```?

Reply:
good catch howard, it should be. will update. thanks.